files/build/scripts/j2

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+update_version=$SONIC_VERSION_CONTROL_COMPONENTS
+version_file=
+if [ -d /sonic ];then
+    version_file=/sonic/files/build/versions/versions-docker
+else
+    version_file=files/build/versions/versions-docker
+fi
```

Comment:
Should we hard coded it to `/usr/local/share/buildinfo/versions/versions-docker` ?